README.md

```diff
@@ -1,92 +1,123 @@
-  ```
+
+* Install and setup **Airflow** (https://airflow.apache.org/docs/stable/installation.html):
+
+  ```bash
   export AIRFLOW_HOME=~/airflow
   pip install apache-airflow
   airflow initdb
   ```
-* Start the server and visit your airflow admin UI
-  ```
+  
+_Note_: On recent versions of Python (3.7+), you may face the following error when executing `airflow initdb`:
+
+```bash
+ModuleNotFoundError: No module named 'typing_extensions'
+```
+
+This can be solved with `pip install typing_extensions`.
+
+* Then, start the server and visit your Airflow admin UI:
+
+  ```bash
   airflow webserver -p 8080
   ```
 
-Now jump into one of the examples.
+By default, the server will be accessible at `http://0.0.0.0:8080/` as shown in the output of the terminal where you ran the previous command.
 
+Now, jump into one of the examples.
 
 ## Examples
 
 ### Example 1: CSV to JSON
 
-In this example we'll run an aircan example to convert a CSV to JSON.
+In this example we'll run an AirCan example to convert a CSV to JSON.
 
-Create the DAG for loading
+Add the DAG to the default directory for Airflow to recognize it:
 
+```bash
+mkdir ~/airflow/dags/
+cp examples/aircan-example-1.csv ~/airflow/
+cp examples/csv_to_json.py ~/airflow/dags/
 ```
-cp aircan/examples/xxx.py ~/airflow/dags/
+
+To see this DAG appear in the Airflow admin UI, you may need to restart the server or launch the scheduler to update the list of DAGs (this may take about a minute or two to update, then refresh the page on the Airflow admin UI):
+
+```bash
+airflow scheduler
 ```
 
 Run this DAG:
 
-* Select this [screenshot]
-* Configure it with a path to ../your/aircan/examples/example1.csv
-* Run it ...
+* Enable the dag in the [admin UI](http://0.0.0.0:8080/) with this toggle to make it run with the scheduler:
+  
+  ![aircan_enable_example_dag_in_scheduler](docs/resources/images/aircan_enable_example_dag_in_scheduler.png)
+* "Trigger" the DAG with this button:
 
-Check the output
+  ![aircan_trigger_example_dag_to_run](docs/resources/images/aircan_trigger_example_dag_to_run.png)
 
-* Locate the output on disk at `~/aircan-example-1.json`
+* After a moment, check the output. You should see a successful run for this DAG:
 
-## Using Aircan DAGs
+  ![aircan_output_example_dag](docs/resources/images/aircan_output_example_dag.png)
 
+* Locate the output on disk at `~/airflow/aircan-example-1.json`
+
+## Using Aircan DAGs
 
 ### Example 2: Local file to CKAN DataStore using the Datastore API
 
 #### Preliminaries: Setup your CKAN instance
 
 We'll assume you have:
 
-* a local CKAN setup and running at https://localhost:5000
-* a dataset (for example, `my-dataset`) with a resource (for example, `my-resource` with the `my-res-id-123` as its `resource_id`).
-* We also need to set up two environment variables for Airflow. Access the Airflow Variable panel and set up `CKAN_SITE_URL` and your `CKAN_SYSADMIN_API_KEY`:
+* a local CKAN setup and running at http://localhost:5000;
```

Comment:
This may seem like a minor issue, but I realized that I couldn't upload to CKAN when setting the environment variable with HTTPS in Airflow because of an SSL error.